hangman.py

```diff
@@ -1,213 +1,304 @@
+      sure that the user puts in a letter!
+
+    * The user should receive feedback immediately after each guess
+      about whether their guess appears in the computer's word.
+    * After each guess, you should display to the user the
+      partially guessed word so far.
+
+    Follows the other limitations detailed in the problem write-up.
+    '''
+    print('''
+ _   _                                           __   _____ 
+| | | |                                         /  | |  _  |
+| |_| | __ _ _ __   __ _ _ __ ___   __ _ _ __   `| | | |/' |
+|  _  |/ _` | '_ \ / _` | '_ ` _ \ / _` | '_ \   | | |  /| |
+| | | | (_| | | | | (_| | | | | | | (_| | | | | _| |_\ |_/ /
+\_| |_/\__,_|_| |_|\__, |_| |_| |_|\__,_|_| |_| \___(_)___/ 
+                    __/ |                                   
+                   |___/                                    
+''')
+    print(f'I am thinking of a word that is {len(secret_word)} letters long.')
+    guesses = 6
+    letters_guessed = []
+    vowels = ('a', 'e', 'i', 'o', 'u')
+    warnings = 3
+    while 1:
+        if is_word_guessed(secret_word, letters_guessed):
+            print(victory(guesses, secret_word))
+            break
+        elif guesses <= 0:
+            print(f'\033[7mYou lost. Guesses exceeded. The secret word was - {secret_word}\033[0m')
+            break
+        else:
+            print(get_guessed_word(secret_word, letters_guessed))
+            print(f'You have {guesses} guesses left.')
+            available = get_available_letters(letters_guessed)
+            print(f'Available letters: {available}')
+            l_c = letter_check(guesses, warnings)
+            letter = l_c[0]
+            guesses = l_c[1]
+            warnings = l_c[2]
+            letters_guessed.append(letter.lower())
+            if letter.lower() in secret_word:
+                print('\033[1mRight guess!\033[0m')
+            elif letter.lower() in vowels and letter.lower() not in secret_word:
+                print('\033[31mWrong guess(\033[0m')
+                guesses -= 2
+            else:
+                print('\033[31mWrong guess(\033[0m')
+                guesses -= 1
+            print('--------------------------------------------')
+
+
+def match_with_gaps(my_word, other_word):
+    '''
+    my_word: string with _ characters, current guess of secret word
+    other_word: string, regular English word
+    returns: boolean, True if all the actual letters of my_word match the
+        corresponding letters of other_word, or the letter is the special symbol
+        _ , and my_word and other_word are of the same length;
+        False otherwise:
+    '''
+    my_word_new = my_word.replace(' ', '')
+    if len(my_word_new) == len(other_word):
+        for i in range(len(my_word_new)):
+            if my_word_new[i] == other_word[i]:
+                continue
+            elif my_word_new[i] == "_" and other_word[i] not in list(my_word_new):
+                continue

```

Comment:
I believe you can avoid using `continue` statement

Reply:
How?